corehq/apps/app_manager/suite_xml.py

```diff
@@ -1673,6 +1673,14 @@ def configure_entry_as_case_list_form(self, form, entry):
         frame_case_not_created.add_command(return_to)
         entry.stack.add_frame(frame_case_not_created)
 
+    def get_case_datums(self, module, form):
```

Comment:
What do you think of calling it `get_case_datums_basic_module` instead of a comment? A bit wordy?